src/setup/cortx_deploy

```diff
@@ -94,13 +101,51 @@ validate_config $CLUSTER_INFO
 
 export PATH=$PATH:/opt/seagate/provisioner/bin
 
-# Apply Cluster Config
-log $INFO "cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c '$CORTX_CONFSTORE'"
-cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c "$CORTX_CONFSTORE";
+# Check system is pre-configured:
+# config apply: check confstore file.
+# cluster bootstrap: check VERSION file is exist and phase and status key from confstore and.
+
+node_id=`cat /etc/machine-id`
+cluster_id=`conf $(yaml_config $CLUSTER_INFO) get "cluster>id"`
+cluster_id=$(echo $cluster_id | tr -d "["\" | tr -d "\"]")
+log $INFO "Starting deployment on node $node_id which is part of cluster $cluster_id."
 
-# Apply Common Config
-log $INFO "cortx_setup config apply -f $(yaml_config $CONFIG_INFO) -c '$CORTX_CONFSTORE'"
-cortx_setup config apply -f $(yaml_config $CONFIG_INFO) -c "$CORTX_CONFSTORE";
+FILE=$(awk -F:// '{print $2}' <<< $CORTX_CONFSTORE)
```

Comment:
We should not assume that logs are stored in the config file. We are planning to move to consul. 
This logic should just work. 

The way we should do is... conf get <some node specific value>. If this value is set this means config is applied.

Reply:
As discussed pre-configured deployment check handled in provisioner.py.
Removed this code from cortx_deploy.